python/ray/tune/integration/comet.py

```diff
@@ -1,28 +0,0 @@
-from ray.air.integrations.comet import CometLoggerCallback as _CometLoggerCallback
-from typing import List
-
-import logging
-
-from ray.util.annotations import Deprecated
-
-logger = logging.getLogger(__name__)
-
-callback_deprecation_message = (
-    "`ray.tune.integration.comet.CometLoggerCallback` "
-    "is deprecated and will be removed in "
-    "the future. Please use `ray.air.integrations.comet.CometLoggerCallback` "
-    "instead."
-)
-
-
-@Deprecated(message=callback_deprecation_message)
-class CometLoggerCallback(_CometLoggerCallback):
-    def __init__(
-        self,
-        online: bool = True,
-        tags: List[str] = None,
-        save_checkpoints: bool = False,
-        **experiment_kwargs
-    ):
-        logging.warning(callback_deprecation_message)
```

Comment:
Instead of removing, should we raise an exception here?

Reply:
Ah yeah, technically it's been deprecated for like 4 releases, so I was going to skip directly to removal, but we can just hard deprecate then remove in 2.6.

So, the schedule is 2 releases of soft deprecation, then 1 release of hard deprecation, then removal?

Reply:
For public APIs, yes! For beta and DeveloperAPIs, 1 release of soft deprecation is sufficient.
I think we should just always do the error raising - people may ignore warnings (especially if scripts are run in jobs/on schedules) and the raise will make sure they can migrate their scripts easily.